ansible/examples/slave.yml

```diff
@@ -167,6 +166,7 @@
           - python-pip
           - python-devel
           - python-virtualenv
+          - mock
```

Comment:
just for the posterities:

https://centos.pkgs.org/7/centos-extras-x86_64/mock-1.2.17-1.el7.centos.noarch.rpm.html

EPEL7's version is newer:

https://centos.pkgs.org/8/epel-x86_64/mock-2.3-1.el8.noarch.rpm.html